src/components/Select/Select.test.tsx

```diff
@@ -45,50 +44,42 @@ describe('<Select />', () => {
 		});
 	});
 
-	it('Should pass on the id', () => {
-		const id = 'test';
-
-		const selectComponent = shallow(<Select options={options} id={id} />);
-
-		const selectElement = selectComponent.find('select');
-
-		expect(selectElement.prop('id')).toEqual(id);
-	});
-
 	it('Should be able to set the disabled state', () => {
-		const selectComponent = shallow(<Select options={options} disabled />);
+		const selectComponent = mount(<Select options={options} disabled />);
 
-		const selectElement = selectComponent.find('select');
+		const selectElement = selectComponent.find('div').at(0);
 
-		expect(selectElement.prop('disabled')).toEqual(true);
+		expect(selectElement.hasClass('c-select--is-disabled')).toEqual(true);
 	});
 
-	it('Should be able to set an initial value', () => {
-		const defaultValue = options[5].value;
-
-		const selectComponent = shallow(<Select options={options} value={defaultValue} />);
-
-		const selectElement = selectComponent.find('select');
-
-		expect(selectElement.prop('value')).toEqual(defaultValue);
-	});
-
-	it('Should call the onChange handler when the select option changes', () => {
-		const onChangeHandler = jest.fn();
-
-		const selectComponent = shallow(<Select options={options} onChange={onChangeHandler} />);
-
-		const selectElement = selectComponent.find('select');
-
-		selectElement.simulate('change', { target: { value: options[3] } });
-
-		expect(onChangeHandler).toHaveBeenCalled();
-		expect(onChangeHandler).toHaveBeenCalledTimes(1);
-		expect(onChangeHandler).toHaveBeenCalledWith(options[3]);
-
-		selectElement.simulate('change', { target: { value: options[6] } });
-
-		expect(onChangeHandler).toHaveBeenCalledTimes(2);
-		expect(onChangeHandler).toHaveBeenCalledWith(options[6]);
-	});
+	// it('Should be able to set an initial value', () => {
```

Comment:
Maybe add a TODO so we don't forget to resolve these commented lines.

Reply:
ik zal ze weg doen.
For the record, i really hate enzyme